src/test/physics.cpp

```diff
@@ -0,0 +1,116 @@
+#include <gtest/gtest.h>
+#include <crepe/ComponentManager.h>
+#include <crepe/api/GameObject.h>
+#include <crepe/api/Rigidbody.h>
+#include <crepe/api/Transform.h>
+#include <crepe/system/PhysicsSystem.h>
+#include <crepe/api/Config.h>
+
+using namespace std;
+using namespace std::chrono_literals;
+using namespace crepe;
+
+class PhysicsTest : public ::testing::Test {
+protected:
+    GameObject* game_object;
```

Comment:
Dit is niet RAII, maar ik betwijfel of het nut heeft om daar energie in te gaan steken want het is slechts een test.